core/cmd/shell.go

```diff
@@ -210,15 +211,18 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg chainlink.G
 
 	capabilitiesRegistry := capabilities.NewRegistry(appLggr)
 
+	retirementReportCache := llo.NewRetirementReportCache(appLggr, ds)
```

Comment:
Does this require the services to start synchronously with application startup? Could the related services be kicked off in the ocr2 job delegate flow instead?